### ISSUE TITLE
Backpacking scratchblocks in forums

### ISSUE THREAD

**Issue by FunctionalMetatable** [ScratchAddons/ScratchAddons#3225]
### Is your request related to a problem? Please describe.

you should be able to backpack scratchblocks in the forums, like you were allowed to for a short timeframe
### Describe the solution you'd like
a way to backpack scratchblocks in the forums

### Describe alternatives you've considered
none

### Additional context
n/a

**Comment by apple502j** [comment]
Note that this requires scratchblocks parser, which I don't think it exists for 3.x yet.

**Comment by PullJosh** [comment]
@towerofnix That would be awesome!

Just a note if this ever does happen: `sb-edit` is awesome, but its outward-facing API is pretty terrible. So if you feel like something is harder than it needs to be, please let me know. Maybe y'all could submit a PR that improves the API...

**Comment by towerofnix** [comment]
@PullJosh Yeah, the outward-facing API was the main barrier that kept me from messing around too much back in the day, admittedly. :P I think it could use some big improvements and that would make adding support for scratchblocks (and thus an addon like this) a lot easier!

(I don't mean to go off-topic here though! :chestnut:)

**Comment by apple502j** [comment]
scratchblocks itself is ambiguous and it's very hard to parse things (ik because i have tried once) - maybe for english it could work. Not so much for other languages, like Japanese where a block without name exists, or in German and whatever languages with different disambiguation rules...

**Comment by medlans** [comment]
> Note that this requires scratchblocks parser, which I don't think it exists for 3.x yet.

What about now?
